lib/transition.js

```diff
@@ -33,6 +36,8 @@ class Transition {
         if (!prefixer || !prefixer.prefixes) continue
 
         for (prefix of prefixer.prefixes) {
+          if (vendorPrefixes && !vendorPrefixes.includes(prefix)) continue
```

Comment:
We can always have array in `vendorPrefixes` (by returning `[]` instead of `false`) and about this extra check.

Reply:
I like the idea of returning ```false``` cause here we need to check whether rule is vendor specific. If it's not - we don't have to check the prefix. By the way I just found out the check itself 
```js
!vendorPrefixes.includes(prefix)
```
works incorrect. Prefix might be ```-webkit- 2009``` for example. I'll fix it.